### PR TITLE
fix(images): Show UI earlier for large multi-page images

### PR DESCRIPTION
**Pull request by mickr**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @mickr has signed the CLA. Thanks for the pull request!

File: src/lib/viewers/image/MultiImageViewer.js [hunk 1]
```diff
@@ -151,8 +163,14 @@ class MultiImageViewer extends ImageBaseViewer {
...
-    /** @inheritdoc */
-    setOriginalImageSize() {
+    /**
+     * Sets the original image width and height on the img element. Can be removed when
+     * naturalHeight and naturalWidth attributes work correctly in IE 11.
+     *
+     * @protected
+     * @return {Promise} A promise that is resolved if the original image dimensions were set.
+     */
+    setOriginalImageSizes() {
```
Comment (jstoffan) [inline src/lib/viewers/image/MultiImageViewer.js]: Any tests needed?

File: src/lib/viewers/image/MultiImageViewer.js [hunk 2]
```diff
@@ -90,6 +91,17 @@ class MultiImageViewer extends ImageBaseViewer {
             .catch(this.handleAssetError);
     }
 
+    /**
+     * Handles the load event for the first image.
+     *
+     * @return {Promise} Promise to load bunch of images
```
Comment (jstoffan) [inline src/lib/viewers/image/MultiImageViewer.js]: Is this accurate?

File: src/lib/viewers/image/__tests__/MultiImageViewer-test.js [hunk 3]
```diff
@@ -232,7 +232,7 @@ describe('lib/viewers/image/MultiImageViewer', () => {
         });
     });
 
-    describe('setOriginalImageSize()', () => {
+    describe('setOriginalImageSizes()', () => {
```
Comment (jstoffan) [inline src/lib/viewers/image/__tests__/MultiImageViewer-test.js]: Should the child tests now invoke `multiImage.setOriginalImageSizes()`, instead?

**Pull request merged**